src/config/entities/__tests__/configuration.ts

```diff
@@ -276,6 +276,7 @@ export default (): ReturnType<typeof configuration> => ({
   },
   spaces: {
     maxSafesPerSpace: faker.number.int({ min: 5, max: 10 }),
+    maxSpaceCreationsPerUser: faker.number.int({ min: 1000, max: 2000 }),
```

Comment:
Nit: This seems a bit high for a test value. Shall we reduce it to `100`?

Reply:
Changed in [b373003](https://github.com/safe-global/safe-client-gateway/pull/2516/commits/b37300317fa7ed80eae48618d255b755fb02caf0)